lib/travis/build/templates/header.sh

```diff
@@ -273,6 +273,22 @@ vers2int() {
   printf '1%03d%03d%03d%03d' $(echo "$1" | tr '.' ' ')
 }
 
+_ensure_ruby_23() {
+  if ! (rvm list | grep 2\.[3-9] 2>&1 >/dev/null); then
```

Comment:
Nitpick: `! (rvm list | grep 2\.[3-9])` -> `rvm list | grep -v 2\.[3-9]`

Reply:
No. Yours would return `true` if any line from `rvm list` output does not match `2\.[3-9]`.

Reply:
Oh, of course. Sorry for the noise.